Resources/ServerInfo/_CP14/Guidebook_EN/JobsTabs/Innkeepers.xml

```diff
@@ -0,0 +1,32 @@
+<Document>
+  # Innkeeping
+  The innkeeper is a role that primarily stays within the town. The responsibilities of an innkeeper include gardening, bartending, and cooking.
+
+  Within the inn, an innkeeper will have access to three main areas:
+
+  ## The Garden
+  The garden, or farm, is an important asset for any chef. A few seedbeds can be found here, along with a well and some basic supplies.
+  To begin farming, locate the supply table in the garden. Usually, a crate will contain a variety of seeds, while a table will hold farming tools like a bucket and hoe. Grab a seed variety and use it on the ground to plant it. Wait a while, then use any sharp tool (such as a sickle or dagger) to harvest the fully grown crop.
```

Comment:
It is also necessary to add here that by growing cotton, you can make bags for carrying vegetables or seeds. To do this, you need to get wood outside the city in advance, cut it into boards with a knife and then make a workbench.